packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -131,6 +142,7 @@ abstract class TextSelectionControls {
     List<TextSelectionPoint> endpoints,
     TextSelectionDelegate delegate,
     ClipboardStatusNotifier clipboardStatus,
+    Offset? lastSecondaryTapDownPosition,
```

Comment:
@justinmc This is a breaking change.

What is the suggested migration?

Reply:
(this breaks `flutter_math` e.g. - I will link the PR)

Reply:
Ah sorry about that.  The migration should be to pass `renderObject.lastSecondaryTapDownPosition`.  See https://github.com/flutter/flutter/pull/73882/files#diff-6f186b82dcb3d864ea23246744ea7a67bcb64d369e0da5495cd0bd0e038f9bd9R600

Reply:
Alright, thanks ❤️